### PR TITLE
Fix: Fixed issue with file preview in column layout

### LINKED ISSUE #12227: Bug: Issue with file preview in column layout
### Description

When in column view mode with the preview pane enabled, clicking on the first file displays its characteristics, folder logo, etc. However, when I click on the next file or folder, nothing appears and I have to click on another folder or in the blank space for the preview to display properly again.

### Steps To Reproduce

Open "Files"
Choose a test category, such as "Documents"
Set the view to "Columns"
Enable the preview pane
Click on a folder, and then on a second one

### Requirements

1. Fix the issue with the preview not displaying correctly when switching from one file/folder to another in column view mode.
2. Ensure that the preview displays smoothly and successfully for each selected file/folder without any interruption.

### Files Version

Version : 2.4.70.0

### Windows Version

Windows 11 22H2 22621.1635

### Log file

No need.

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #12227 

This PR also handles a null exception 

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [x] Did you implement any design changes to an existing feature?
   - [x] Was this change approved?
- [x] Are there any other steps that were used to validate these changes? (I used @ZLAllen steps)
   1. Open "Files"
   2. Select "Documents"
   3. Change View Layout to "Columns"
   4. Enable Preview Panel
   5. Click any items in the first level, the detail shows up correctly in the preview.
   6. Select a folder and click on an item at the second level, the preview is not rendered. Select a different item or lose focus of the current selection and reselect the item will cause it to render correctly in the preview.

### CODE REVIEW

File: src/Files.App/Helpers/DriveHelpers.cs [hunk 1]
```diff
@@ -136,9 +136,13 @@ public static Data.Items.DriveType GetDriveType(System.IO.DriveInfo drive)
...
-			=> (StorageItemThumbnail)await FilesystemTasks.Wrap(()
-				=> folder.GetThumbnailAsync(ThumbnailMode.SingleItem, 40, ThumbnailOptions.UseCurrentScale).AsTask()
+		public static async Task<StorageItemThumbnail?> GetThumbnailAsync(StorageFolder folder)
+		{
+			return folder is null
+				? null
+				: (StorageItemThumbnail)await FilesystemTasks.Wrap(()
+					=> folder.GetThumbnailAsync(ThumbnailMode.SingleItem, 40, ThumbnailOptions.UseCurrentScale).AsTask()
 			);
+		}
```
Comment (hishitetsu) [inline src/Files.App/Helpers/DriveHelpers.cs:146]: This is an unrelated fix.

Comment (ferrariofilippo) [reply src/Files.App/Helpers/DriveHelpers.cs:146]: I added it because I was getting that exception every time I started Files and it was annoying. Anyway I'll remove it if you think so

Comment (hishitetsu) [reply src/Files.App/Helpers/DriveHelpers.cs:146]: This will be fixed by #12286.

**Review by hishitetsu — APPROVED** [review]
LGTM

**Review by hecksmosis — APPROVED** [review]
LGTM

**Comment by yaira2** [comment]
Thank you
